src/components/CustomerDetails/InputNricSection.tsx

```diff
@@ -14,7 +14,7 @@ const styles = StyleSheet.create({
   },
   horizontalRule: {
     borderBottomColor: color("grey", 30),
-    marginHorizontal: -size(3),
+    marginHorizontal: -size(0.5),
```

Comment:
What's this change for?

Reply:
This is to ensure that the horizontal rule looks exactly like the mockup because previously the line didn't have a space within the card.
<img width="267" alt="Screenshot 2020-06-17 at 11 18 35 AM" src="https://user-images.githubusercontent.com/38589870/84851256-6125d200-b08c-11ea-9e56-e3f9b3771579.png">

Reply:
Ok so this affects the existing screens, and not for the new screens?

Reply:
Yep, we edited the scan nric part too!